docs/designing-apis/getting-started.md

```diff
@@ -0,0 +1,36 @@
+---
+tags: []
+---
+
+# Modeling Introduction 
+
+
+## What is API Design? 
+
+Designing (also known as Modeling) your API involves describing all of the inputs and outputs across the footprint of your entire service. Your API design will answer questions like: 
+
+- What are the different resources and operations available in your API?
+- How does your API authenticate requests? 
+- What are the different data models associated with your service? 
+- How does your API handle errors? 
+
+## How does it fit into Stoplight? 
+
+The Stoplight design module is where you and your team will maintain the single source of truth that describes the inputs and outputs of your API. 
+
+Once you have your API described in the Stoplight design module, you can: 
+
+- Publish all or part of your API 
+- Create API tests from your designs 
```

Comment:
Can you? Is this talking about the yet-to-be-built Scenarios replacement, or talking about general "you can use these for contract testing if you want" type of stuff.

Reply:
this section was pulled from our previous content as filler, what features do you think we should highlight here? what will you be able to do with explorer? is it worth mentioning in Studio docs?

Reply:
Ah gotcha, well we removed Scenarios so lets hide this. We can add a thing about spectral after mocking on line 26:

- Get feedback in the form of validation and style warnings